README.md

```diff
@@ -68,134 +62,54 @@ then as soon as your PR is merged into master, it will be merged into the curren
-`archive` field to true and the `archive_date` field to the current date, and the `archive_search_refinement`
-to the previous release version (in this case `V1.3`).
-
-1. In the **previous release's** branch (in this case `release-1.3`), edit the file `config.toml`. Set the
-`disableAliases` field to `false`.
-
-1. Commit the previous edits to your local git repo and push the **previous release's** branch to GitHub.
-
-1. In the **archive** branch, rebase the branch to have all changes from the current release. In this case,
-all changes from the `release-1.4` branch.
-
-1. Commit the previous edits to your local git repo and push the **archive** branch to GitHub.
-
-1. Wait a while (~20 minutes) and browse archive.istio.io to make sure everything looks good.
-
-##### Updating preliminary.istio.io
-
-1. In the **master** branch, edit the file `data/args.yml`. Set the `version` and `full_version` fields to have the version
-of the next Istio release, and `previous_version` to be the version of the previous release. In this case, you would set the fields to
-"1.5", "1.5.0", and "1.4" respectively.
-
-1. In the **master** branch, edit the file `data/args.yml`. Set the
-`source_branch_name` and `doc_branch_name` fields to `master`.
-
-1. In the **master** branch, edit the file `Makefile.core.mk`. Set the variable `SOURCE_BRANCH_NAKE` to
-`master`.
-
-1. Run `make update_all` in order to retrieve the latest material from the source repositories.
-
-1. Commit the previous edits to your local git repo and push the **master** branch to GitHub.
-
-1. Wait a while (~5 minutes) and browse preliminary.istio.io to make sure everything looks good.
+    case, the site URL would be archive.istio.io/v1.6/*. Set the label of this site to the name of the
+    facet item created above (V1.6 in this case).
 
 ### Creating a patch release
 
 A few days before the patch release, the release managers should notify the Doc WG that the release
 is built and is starting it's long running qualification test. At this time, move the doc automation
-tests to use the new release to verify automated doc testing passes. To move to a new release
-(make sure you are in the patch's release branch):
-
-1. `go get istio.io/istio@1.X.Y`
-
-1. `go mod tidy`
-
-1. Create a PR with the go.* changes.
+tests to use the new release to verify automated doc testing passes. 
 
-Creating a new patch release involves modifying a few files:
+To create a new patch release, you need to
 
-1. Create the release note for the release by adding a markdown file in
-`content/en/news/<YEAR>/1.X.Y/index.md`, where 1.X.Y is the name of the release. This is where
-you describe the changes in the release.
+1. Run `make release-A.X.Y`, where `A.X.Y` is the name of the release. If it is a private security patch, run `PRIVATE_PATCH=true make release-A.X.Y`, which will prevent the branch from grabbing the latest docs.
```

Comment:
grabbing the latest docs and updating the istio reference for testing.

We probably still want a step to do that after the private release is ready and made public. So maybe we want to add some words around running again without the PRIVATE_PACTH=true when the release is made. Actually, since we have everything inside the PRIVATE_PATCH (we removed the templating), maybe we can change the code to remove PRIVATE_PATCH and just have the instructions say to run it when a public build qualification is started (as directed by the release managers) or after a private release is made.

Reply:
Not sure if I've understood it clearly... if PRIVATE_PATCH is removed then "after a private release is made," go mod and go tidy will still be run, which sounds like opposite of a previous [comment](https://github.com/istio/istio.io/pull/7601#discussion_r445184673).

Reply:
My thought was that we were left with only two basic items in this flow, the go.* update and the reference document update after removing the templates. I looked again today and there is actually a third part of the update, the data/args.yml update.

In your testing, do you know if we can update the args.yml before we add the files in the news directory and have a PR pass its tests (I know there is an issue if you add the files without updating the args.yml)? I suspect the answer is that we can't, and we need to keep the news files and args.yml change within the same PR.

If we have to merge the args.yml change with the news, then I think we can remove that piece from automation and just let the release managers worry about that. This leaves us with the two pieces: go.* and reference docs. It then makes sense to remove the PRIVATE_PATCH check and simply have the target run ~3 days before the release if it's built publicly or after the release is publish if built privately.

If we can merge the args.yml change early without breaking things, then that could help with cherrypicking the news files. @frankbu Feedback from you as well, but should the release managers create the release notes in _master_ and then cherrypick to the _release_ branch and update the args.yml in that PR, or should they make the change in the _release_ branch and then manually cherrypick the changes back to _master_ without the args.yml change? In the past, the former seemed to be common, but recently, the later has been more common (I think because that limit the bottleneck for PR merging to one release of istio.io on publishing day and less likely to need a force merge).

Reply:
Comment from @brian-avery on ordering: `The process we decided on in the PSC is release first and then master given that we need to make sure it works in the release branch and not block on that, and we can get master merged after that.`

Reply:
Per my test it seems that args.yml *can* be merged early without breaking things (#7634). So I guess we can run this as it is now. After the release note is completed in the release branch, both changes can then be cherry-picked to master.

Reply:
The problem I noted with merging early in your other PR is that it updates the latest version shown on the bottom of the page (which isn't released yet). So we shouldn't be updating that until the release is being published (at the same time as the release/security notes).

So I think we have a go.* update along with the reference doc update that runs ~3 days before the release for public releases (when asked by the person heading up the release), or after the release is published and public repos updated in the case of a private release.  In the case of the private built release we could ask the release them to do the run the automated job and then let that get cherrypicked on publish day. Actually, it might just be best to have the release team run and submit the PR to the public/private repo as needed for the updates.

There is another piece of work which is the doc updates at publish time (args.yml and notes). That just comes from the release side. First to the `release` branch and then manual cherrypick of the notes to `master`.

Reply:
Sounds good. I made changes to patch_release.sh and updated [README](https://github.com/istio/istio.io/pull/7601/files#diff-04c6e90faac2675aa89e2176d2eec7d8R104). Does this look good to you?